tests/unit/recon/test_schema_comparator.py

```diff
@@ -100,3 +101,36 @@ def test_schema_comparison_failure(metadata_row_factory):
     schema_comparator = StandardSchemaComparator(metadata_retriever)
     actual_comparison_result = schema_comparator.compare_schema(source, target)
     assert actual_comparison_result == expected_comparison_result
+
+
+@pytest.mark.parametrize(
+    "source_column, target_column, case_sensitive, expected_pass",
+    [
+        ("column1", "columnx", True, False),
+        ("column1", "column1", True, True),
+        ("column1", "Column1", True, False),
+        ("column1", "Column1", False, True),
+        ("CoLuMn1", "cOlUmN1", True, False),
+        ("CoLuMn1", "cOlUmN1", False, True),
+    ],
+)
+def test_schema_comparison_case(metadata_row_factory, source_column, target_column, case_sensitive, expected_pass):
```

Comment:
Nice! Only enable linting:

```suggestion
def test_schema_comparison_case(metadata_row_factory, source_column, target_column, case_sensitive, expected_pass) -> None:
```